### ISSUE TITLE
[Enhancement] Use Fonts instead of images

### ISSUE THREAD

**Issue by GreenLunar** [beniz/seeks#14]
Just as GitHub and others do.  Be it with characters of custom special set of fonts or otherwise.

For example (not custom):
Instead of image `/src/plugins/websearch/public/images/seeks_croix_light_10x10.png` Use `⨉` or `✕` or `✖` or `✗` or `✘` and color it in `#F9778C`. `<p><font color="#F9778C">✕</font></p>`.

Using custom font:

You may find brand icons of search engines at:
http://richstyle.org/font.php (**this set seems like a good compilation**)
http://www.fontspace.com/ptocheia/socialicons
http://www.fontspace.com/matt-grey-design/social-networking-icons
http://fontawesome.io/icons/#brand

Anything else:
http://richstyle.org/font.php
http://fortawesome.github.io/Font-Awesome/icons/
http://fontawesome.io/icons/#directional
http://fontawesome.io/icons/#text-editor
https://icomoon.io/

Some alternatives to images...

Search web pages
http://fontawesome.io/icon/list-alt
http://fontawesome.io/icon/globe
Images
http://fontawesome.io/icon/camera/
Videos
http://fontawesome.io/icon/youtube-play
Tweets
http://fontawesome.io/icon/comments/
Help
http://fontawesome.io/icon/life-ring

**Comment by beniz** [comment]
Can you PR changes for this ?

**Comment by GreenLunar** [comment]
No, unless you would help me with some guidance. I never dealt with @face (for custom font) in CSS.

Would you like to correspond via Email?

**Comment by beniz** [comment]
@GreenLunar sorry I cannot help with anything related to Web dev such as CSS or HTML, just not my field.
